nixos/release-small.nix

```diff
@@ -85,6 +85,8 @@ in rec {
 
   tested = let
     onSupported = x: map (system: "${x}.${system}") supportedSystems;
+    onSystems = systems: x: map (system: "${x}.${system}")
+        (pkgs.lib.intersectLists systems (supportedSystems ++ limitedSupportedSystems));
```

Comment:
No `limitedSupportedSystems` in this file so far.

Reply:
So just this line?
```nix
        (pkgs.lib.intersectLists systems supportedSystems);
```

Reply:
Hah, yeah. I was in a hurry and didn't test this properly. Fix pushed.